tests/expr/fbinary/test-add.py

```diff
@@ -21,10 +21,39 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 # IN THE SOFTWARE.
 #-------------------------------------------------------------------------------
+import pytest
 from datatable import dt, f
 from tests import assert_equals
 
 
+#-------------------------------------------------------------------------------
+# Errors
+#-------------------------------------------------------------------------------
+
+@pytest.mark.parametrize("type", [dt.obj64])
+def test_add_error_wrong_one_type(type):
+    DT = dt.Frame(A=[[None]*100]/type, B=[range(100)]/dt.int32)
```

Comment:
That's a very strange frame... Have you looked at it?
Anyways, the easiest way to create a frame with given types is either
```
DT = dt.Frame(A=[]/dt.obj64, B=[]/dt.int32)
```
or
```
DT = dt.Frame(names=["A", "B"], stypes=[dt.obj64, dt.int32])
```

Reply:
Indeed, that was probably a typo at some point, not sure how it went through. Though it did created a frame with the needed types, I will simplify it, but we may want to look at what really happened here.

Reply:
Ah, I guess it just converted the `[range(100)]` to one `int32` element, and `[[None]*100]` to one `obj64` element, due to `[]`. Will fix this anyways.